libraries/app/application.cpp

```diff
@@ -463,6 +463,7 @@ namespace detail {
             wild_access.allowed_apis.push_back( "network_broadcast_api" );
             wild_access.allowed_apis.push_back( "history_api" );
             wild_access.allowed_apis.push_back( "crypto_api" );
+			wild_access.allowed_apis.push_back( "asset_api" );
```

Comment:
Note that some of the queries given by this API could be quite slow (iterating all balances in the blockchain, for instance) so giving unchecked public access to this API might be a potential denial of service vulnerability. Or it may not be that big an issue. Might be worth testing before putting into production.